packages/api/models/collections.js

```diff
@@ -44,9 +45,9 @@ class Collection extends Manager {
   }
 
   async create(item) {
-    // write the record to S3
-    const key = `${process.env.stackName}/collections/${item.name}.json`;
-    await S3.put(process.env.internal, key, JSON.stringify(item));
+    const collectionConfigStore =
+      new CollectionConfigStore(process.env.internal, process.env.stackName);
+    await collectionConfigStore.put(item.name, item);
```

Comment:
We now have one central piece of code that manages the storage and retrieval of collection configs in S3.